appclient/src/main/resources/schema/application-client_9.xsd

```diff
@@ -0,0 +1,307 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns="http://www.w3.org/2001/XMLSchema"
+            targetNamespace="https://jakarta.ee/xml/ns/jakartaee"
+            xmlns:jakartaee="https://jakarta.ee/xml/ns/jakartaee"
+            xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+            elementFormDefault="qualified"
+            attributeFormDefault="unqualified"
+            version="9">
+  <xsd:annotation>
+    <xsd:documentation>
+
+      Copyright (c) 2009, 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
The dates look correct in that they match https://jakarta.ee/xml/ns/jakartaee/application-client_9.xsd.  Since we didn't make any significant change to application-client_9.xsd I agree that the date doesn't need to updated to 2021.

Reply:
👍 My thinking is we are just shipping the standard files, so we don't touch them.